### PR TITLE
[SD-4295]-(fix) - Allow word to add in dictionary with different case…

### PR DESCRIPTION
**Pull request by sjunaid**
… versions (In Dictionary Settings)

### CODE REVIEW

File: scripts/superdesk-dictionaries/views/dictionary-config-modal.html [hunk 1]
```diff
@@ -61,7 +61,6 @@ <h3 ng-show="dictionary._id" translate>Edit Dictionary</h3>
                 </div>
                 <button class="btn flex-field no-shrink"
                     type="submit"
-                    ng-disabled="!isNew"
```
Comment (petrjasek) [inline scripts/superdesk-dictionaries/views/dictionary-config-modal.html]: might be missing something, but how do you detect if word is in dict already?

File: scripts/superdesk-dictionaries/dictionaries.js [hunk 2]
```diff
@@ -280,25 +280,22 @@
...
+                        words = [],
+                        word;
+                    for (var i = 0; i < length; i++) {
+                        word = searchWords[i];
+                        if ($scope.dictionary.content[word] > 0 && isPrefix(search, word)) {
+                            words.push(word);
                         }
                     }
+                    words.sort();
+                    $scope.words = _.uniq(words);
```
Comment (petrjasek) [inline scripts/superdesk-dictionaries/dictionaries.js]: uniq here is not needed, there is no duplication in dicts

**Comment by Aca-jov** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**